File: AndroidStudio/app/src/main/java/org/techtown/hello/MainActivity.java
 package org.techtown.hello;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.net.Uri;
import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

 public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void Button1Clicked(View v){
        Toast.makeText(getApplicationContext(),"네이버 버튼이 눌렸어요.", Toast.LENGTH_LONG).show();
        Intent myIntent = new Intent(Intent.ACTION_VIEW, Uri.parse("http://m.naver.com"));
        startActivity(myIntent);
    }

    public void Button2Clicked(View v){
        Toast.makeText(getApplicationContext(),"전화 걸기 버튼이 눌렸어요.", Toast.LENGTH_LONG).show();
        Intent myIntent = new Intent(Intent.ACTION_VIEW, Uri.parse("tel:010-2734-1966"));
        startActivity(myIntent);
    }

    public void Button3Clicked(View v){
        Intent intent = new Intent(getApplicationContext(), MenuActivity.class);
        startActivity(intent);
    }
}

File: AndroidLab2/project_module/src/main/java/com/example/project_module/MainActivity.kt
package com.example.project_module

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity


class MainActivity : AppCompatActivity() {
    private lateinit var btn_go_walk : Button
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btn_go_walk = findViewById(R.id.go_walk)
        btn_go_walk.setOnClickListener(){
            val intent = Intent(this,WalkActivity::class.java)
            startActivity(intent)
        }
    }
}